packages/wrangler/templates/middleware/middleware-ensure-req-body-drained.ts

```diff
@@ -0,0 +1,14 @@
+import type { Middleware } from "./common";
+
+const drainBody: Middleware = async (request, env, _ctx, middlewareCtx) => {
+	try {
+		return await middlewareCtx.next(request, env);
+	} finally {
+		if (!request.bodyUsed) {
+			_ctx.waitUntil(request.arrayBuffer());
+		}
+	}
+};
+
+export default drainBody;
+export const wrap = undefined;
```

Comment:
```suggestion

```

Reply:
I copied another middleware file. I assume it was required for the module interface of a "middleware". Not necessary then?

Reply:
It shouldn't be, unless something has changed since I added the `wrap` export. It _should_ be optional